meta-balena-common/recipes-bsp/grub/grub-conf/grub.cfg_external

```diff
@@ -0,0 +1,12 @@
+# Automatically created by OE
+serial --unit=0 --speed=115200 --word=8 --parity=no --stop=1
+insmod ext2
+insmod configfile
+source /grub/grub_extraenv
+default=boot
+timeout=3
+
+menuentry 'flash'{
+search --set=root --label flash-rootA
+linux /boot/bzImage root=LABEL=flash-rootA flasher rootwait ${extra_os_cmdline}
```

Comment:
Should `bzImage" be parameterized to `KERNEL_IMAGETYPE`?

Reply:
Good catch, yes, it should.